client/src/components/entities/LinkedEntitiesByItemType.tsx

```diff
@@ -17,6 +17,8 @@
  */
 
 import * as React from "react";
+import { ReactNode, useRef } from "react";
```

Comment:
Let's keep a single import from `"react"`

```suggestion
import React,  { ReactNode, useRef } from "react";
```